src/rustdoc-json-types/lib.rs

```diff
@@ -473,6 +473,7 @@ pub enum GenericParamDefKind {
         type_: Type,
         default: Option<String>,
     },
+    Effect,
```

Comment:
If you're changing the format, could you increase the `FORMAT_VERSION` constant at the top of the file by 1.